tsconfig.json

```diff
@@ -4,7 +4,8 @@
         "target": "ES2017",
         "declaration": true,
         "module": "commonjs",
-        "strict": true
+        "strict": true,
+        "skipLibCheck": true
```

Comment:
Any reasons for doing this?

Reply:
1. although, using `strict: true` in typescript project, makes code a bit tougher to fix but in the long run that does make sense and saves you from a lot of debugging/error-solving sessions
2. there is a clear instructions to [use strict](https://github.com/colinhacks/zod#installation)
3. I guess that is because zod use object schema to reflect typescript types, so strict will make it easier to write less erroneous code